### PR TITLE
Use formmated custom logger, fix bug parsing env private key

### PR DESCRIPTION
**Pull request by alexstolr**

### CODE REVIEW

**Review by IlyaVi — APPROVED** [review]

**Review by IlyaVi — APPROVED** [review]

**Review by axelrod-blox — APPROVED** [review]

**Review by IlyaVi — APPROVED** [review]

File: src/modules/clusters/cli/cluster.transformer.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import TimeAgo from 'javascript-time-ago';
 import en from 'javascript-time-ago/locale/en';
+import {CustomLogger} from "@cli/shared/services/logger.service";
```
Comment (meshin-dev) [inline src/modules/clusters/cli/cluster.transformer.tsx:3]: If you're using webstorm, add extension to support .editorconfig and also enable eslint --fix on save. 
For example, this import should have spaces before and after class name

File: src/modules/clusters/cli/cluster.transformer.tsx [hunk 2]
```diff
@@ -80,7 +82,7 @@ export const transformClusterData = (items, extra: any) => {
       return order.indexOf(a.status.text) - order.indexOf(b.status.text);
     });
   } catch (e) {
-    console.log(e);
+    logger.error(e);
```
Comment (meshin-dev) [inline src/modules/clusters/cli/cluster.transformer.tsx:85]: Add more descriptive explanation, showing that it was attempt to trnsform cluster data. not just raw error

File: src/modules/earnings/cli/earning.transformer.tsx [hunk 3]
```diff
@@ -1,9 +1,12 @@
+import {CustomLogger} from "@cli/shared/services/logger.service";
```
Comment (meshin-dev) [inline src/modules/earnings/cli/earning.transformer.tsx:1]: same in all files with imports

File: src/modules/earnings/cli/earning.transformer.tsx [hunk 4]
```diff
@@ -16,7 +19,7 @@ export const transformEarningData = items => {
       });
     }
   } catch (e) {
-    console.log(e);
+    logger.error(e);
```
Comment (meshin-dev) [inline src/modules/earnings/cli/earning.transformer.tsx:22]: more details

File: src/services/worker/cron/fetch.cron.ts [hunk 5]
```diff
@@ -1,18 +1,20 @@
...
+  private readonly _logger = new CustomLogger(FetchCron.name);
   constructor(private _fetchTask: FetchTask) {}
 
   @Cron(CronExpression.EVERY_SECOND)
   async fetchNewValidators(): Promise<void> {
     try {
       await this._fetchTask.fetchAllEvents();
     } catch (e) {
-      console.log(e);
+      this._logger.error(e);
```
Comment (meshin-dev) [inline src/services/worker/cron/fetch.cron.ts:17]: same

File: src/services/worker/cron/liquidation.cron.ts [hunk 6]
```diff
@@ -1,18 +1,20 @@
...
+  private readonly _logger = new CustomLogger(LiquidationCron.name);
   constructor(private _liquidationTask: LiquidationTask) {}
 
   @Cron(CronExpression.EVERY_10_SECONDS)
   async liquidate(): Promise<void> {
     try {
       await this._liquidationTask.liquidate();
     } catch (e) {
-      console.log(e);
+      this._logger.error(e);
```
Comment (meshin-dev) [inline src/services/worker/cron/liquidation.cron.ts:17]: same

File: src/services/worker/worker.tsx [hunk 7]
```diff
@@ -42,22 +43,25 @@ async function bootstrapApi() {
...
 
   const confService = app.select(WorkerModule).get(ConfService);
-
   const port = confService.getNumber('PORT') || 3000;
   await app.listen(port);
 
-  // eslint-disable-next-line no-console
-  console.info(`WebApp is running on port: ${port}`);
+  app.useLogger(logger);
+  logger.log('API is active');
```
Comment (meshin-dev) [inline src/services/worker/worker.tsx:50]: This is useless log

File: src/shared/services/web3.provider.ts [hunk 8]
```diff
@@ -119,6 +115,8 @@ export class Web3Provider {
       genesisBlock: jsonCoreData.genesisBlock,
     };
 
+
+
```
Comment (meshin-dev) [inline src/shared/services/web3.provider.ts:119]: Too many empty lines

File: src/shared/services/web3.provider.ts [hunk 9]
```diff
@@ -131,9 +129,13 @@ export class Web3Provider {
           hash: new Web3().utils.keccak256(error).substring(0, 10),
         });
       }
+
```
Comment (meshin-dev) [inline src/shared/services/web3.provider.ts:132]: Not necessary to have this empty line

File: src/shared/services/web3.provider.ts [hunk 10]
```diff
@@ -149,6 +151,16 @@ export class Web3Provider {
     return this.contract.abiViews as any;
   }
 
+  async printConfig(){
+
```
Comment (meshin-dev) [inline src/shared/services/web3.provider.ts:155]: Empty line

**Review by meshin-dev — CHANGES_REQUESTED** [review]

File: src/modules/clusters/cli/cluster.transformer.tsx (hunk 1 above)
Comment (alexstolr) [reply src/modules/clusters/cli/cluster.transformer.tsx:3]: V

File: src/modules/clusters/cli/cluster.transformer.tsx (hunk 2 above)
Comment (alexstolr) [reply src/modules/clusters/cli/cluster.transformer.tsx:85]: V

File: src/modules/earnings/cli/earning.transformer.tsx (hunk 3 above)
Comment (alexstolr) [reply src/modules/earnings/cli/earning.transformer.tsx:1]: V

File: src/modules/earnings/cli/earning.transformer.tsx (hunk 4 above)
Comment (alexstolr) [reply src/modules/earnings/cli/earning.transformer.tsx:22]: V

File: src/services/worker/worker.tsx (hunk 7 above)
Comment (alexstolr) [reply src/services/worker/worker.tsx:50]: V

File: src/shared/services/web3.provider.ts (hunk 9 above)
Comment (alexstolr) [reply src/shared/services/web3.provider.ts:132]: V

File: src/shared/services/web3.provider.ts (hunk 8 above)
Comment (alexstolr) [reply src/shared/services/web3.provider.ts:119]: V

File: src/shared/services/web3.provider.ts (hunk 10 above)
Comment (alexstolr) [reply src/shared/services/web3.provider.ts:155]: It's not there already

**Comment by alexstolr** [comment]
@meshin-dev I addresses all your comments, please re-review

**Review by meshin-dev — APPROVED** [review]

**Pull request merged**